tests/testLMSimpleShape.py

```diff
@@ -0,0 +1,228 @@
+        self.assertClose(moments1.getCore().getIyy(), moments2.getCore().getIyy(), atol = .25)
+        self.assertClose(moments1.getCenter().getX(), moments2.getCenter().getX(), atol = .1)
+        self.assertClose(moments1.getCenter().getY(), moments2.getCenter().getY(), atol = .1)
+
+    #   Our only real quantitative test is to test the nGauss=1 EMPsfApprox again the reference.
+    def testLMSimpleShapeNGauss1(self):
+        config = self.makeConfig(self.algName, psfName='meas_extensions_ngmix_EMPsfApprox', nGauss=1)
+        source, msf2 = self.runShape(config, self.algName, self.exposure, model='gauss')
+        source, msf1 = self.runReferenceShape(self.exposure)
+        moments1 = msf1.evaluate().computeMoments()
+        moments2 = msf2.evaluate().computeMoments()
+        self.assertClose(moments1.getCore().getIxx(), moments2.getCore().getIxx(), atol = .1)
+        self.assertClose(moments1.getCore().getIxy(), moments2.getCore().getIxy(), atol = .1)
+        self.assertClose(moments1.getCore().getIyy(), moments2.getCore().getIyy(), atol = .1)
+        self.assertClose(moments1.getCenter().getX(), moments2.getCenter().getX(), atol = .1)
+        self.assertClose(moments1.getCenter().getY(), moments2.getCenter().getY(), atol = .1)
+
+    #   Just test to be sure this runs without errors
+    def testLMSimpleShapeNGauss2(self):
+        config = self.makeConfig(self.algName, psfName='meas_extensions_ngmix_EMPsfApprox', nGauss=2)
+        source, msf = self.runShape(config, self.algName, self.exposure, model='gauss')
+        self.assertEqual(source.get(self.algName + "_flag"), False)
+
+    #   Just test to be sure this runs without errors
+    def testLMSimpleShapeNGauss3(self):
+        config = self.makeConfig(self.algName, psfName='meas_extensions_ngmix_EMPsfApprox', nGauss=3)
+        source, msf = self.runShape(config, self.algName, self.exposure, model='gauss')
+        self.assertEqual(source.get(self.algName + "_flag"), False)
+
+    #   Just test to be sure this runs without errors
+    def testLMSimpleShapeDev(self):
+        config = self.makeConfig(self.algName)
+        source, msf = self.runShape(config, self.algName, self.exposure, model='dev')
+        self.assertEqual(source.get(self.algName + "_flag"), False)
+
+    #   Just test to be sure this runs without errors
+    def testLMSimpleShapeExp(self):
+        config = self.makeConfig(self.algName)
+        source, msf = self.runShape(config, self.algName, self.exposure, model='exp')
+        self.assertEqual(source.get(self.algName + "_flag"), False)
+
+
+def suite():
+    """Returns a suite containing all the test cases in this module."""
+    utilsTests.init()
+
+    suites = []
+    suites += unittest.makeSuite(ShapeTestCase)
+    suites += unittest.makeSuite(utilsTests.MemoryTestCase)
+
+    return unittest.TestSuite(suites)
+
+
+def run(exit = False):
+    """Run the tests"""
+    utilsTests.run(suite(), exit)
```

Comment:
Please use the new-style way of running tests: https://developer.lsst.io/coding/python_testing.html